src/main/java/uk/ac/ebi/tsc/portal/config/WebConfiguration.java

```diff
@@ -22,6 +22,9 @@
     @Value("#{'${be.friendly.origins}'.split(',')}")
     List<String> origins;
 
+    @Value("${ecp.aai.authority}")
+    public String adminAuthority;
+
```

Comment:
Not the best place for it. What about https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-java-bean-binding instaed (so capturing common properties with the same prefix into a Java bean?)
Also names are not the best, could be more descriptive:
`ecp.roles.deployments.view` and `viewDeploymentsRole` instead of 
```
 @Value("${ecp.aai.authority}")
    public String adminAuthority;
```

Reply:
Renaming and bean binding has been done.